local-cli/link/link.js

```diff
@@ -76,31 +74,40 @@ const linkDependencyAndroid = (androidProject, dependency) => {
   });
 };
 
-const linkDependencyWindows = (windowsProject, dependency) => {
-
-  if (!windowsProject || !dependency.config.windows) {
-    return null;
-  }
-
-  const isInstalled = isInstalledWindows(windowsProject, dependency.config.windows);
-
-  if (isInstalled) {
-    log.info(chalk.grey(`Windows module ${dependency.name} is already linked`));
-    return null;
-  }
-
-  return pollParams(dependency.config.params).then(params => {
-    log.info(`Linking ${dependency.name} windows dependency`);
-
-    registerDependencyWindows(
-      dependency.name,
-      dependency.config.windows,
-      params,
-      windowsProject
-    );
-
-    log.info(`Windows module ${dependency.name} has been successfully linked`);
-  });
+const linkDependencyPlatforms = (platforms, project, dependency) => {
+  const ignorePlatforms = ['android', 'ios'];
```

Comment:
Is there any reason why `android` and `ios` can't have the same code path?

Reply:
They can, I just wanted to keep the diff small.

Reply:
#WontFix - There's currently some custom linking logic related to `podfile` for iOS.  Let's do these changes in two phases:
1. Remove `windows` and support platform extensions.
2. Treat iOS and Android as any platform extension.